multimodal/src/autogluon/multimodal/__init__.py

```diff
@@ -6,4 +6,4 @@
 from . import constants, data, models, optimization, predictor, utils
 from .matcher import MultiModalMatcher
 from .predictor import AutoMMPredictor, MultiModalPredictor
-from .utils import download
+from .utils import download, semantic_search
```

Comment:
Let's still put `semantic_search` in `utils` and do not put in the main API.

Reply:
Done.